tests/engines/registries/unit/services/share-search-test.ts

```diff
@@ -30,6 +30,13 @@ function getSearchResponse(identifiers?: string[]) {
                     sources: ['OSF', 'OSF Registries'],
                     subjects: [],
                     subject_synonyms: [],
+                    osf_related_resource_types: {
+                        data: true,
+                        papers: true,
+                        analytic_code: false,
+                        materials: false,
+                        supplements: false,
+                    },
```

Comment:
Does this need to be in the `extra` object as well?

Reply:
SHARE pull it into the top-level dict in the elastic results: https://github.com/CenterForOpenScience/SHARE/pull/793/files#diff-3fd6a02be08431191d9e59f90d0f374c0541c2c13b8b16809d34809211e1eb63R125